ckanext/data_qld/commands.py

```diff
@@ -10,54 +12,78 @@
 _and_ = sqlalchemy.and_
 
 
-class MigrateExtras(CkanCommand):
-    """Migrates legacy field values that were added as free extras to datasets to their schema counterparts.
+class DataQld(CkanCommand):
+    """
+    Data Qld custom commands
+
+    Usage:
+        paster data_qld
```

Comment:
Would be good to document the possible commands here, but as it's for internal use, not critical.

Reply:
i guess we need to add all the command and i could add few lines for the usage